web/scenes/Portal/Teams/TeamId/Apps/AppId/Configuration/AppTopBar/SubmitAppModal/index.tsx

```diff
@@ -103,20 +106,14 @@ export const SubmitAppModal = (props: SubmitAppModalProps) => {
         setOpen(false);
       } catch (error: any) {
         console.error("Submit App Modal Failed: ", error);
-        let errorMessage = "Failed to submit app for review";
```

Comment:
Why was this removed?

Reply:
Knowing they're missing localisation is good to know

Reply:
We're letting them know a couple lines up with `Localisation not set for all languages`, and it's not possible to access error messages from form actions, so this would never fire anyway